### ISSUE TITLE
QuoteMessage does not exist in the new library

### ISSUE THREAD

**Issue by Mahdi-Roozbahani** [Petersoj/alpaca-java#89]
I noticed that QuoteMessage is removed in your latest update version 7.0.

**Comment by Petersoj** [comment]
Yes there was some refactoring of names and such with the Data v2 API in `7.0`. Check the README for updates names. [Here](https://github.com/Petersoj/alpaca-java/blob/master/README.md#Quotes) is the Quotes endpoint example and [here](https://github.com/Petersoj/alpaca-java/blob/master/README.md#Realtime-Market-Data) is the Quotes realtime feed example.

**Comment by Mahdi-Roozbahani** [comment]
Hi Peter,

I am specifically looking into `MarketDataListener streamListener = new MarketDataListenerAdapter`. I realized that you still have `QuoteMessage `there. I also notice that `OrderTimeInForce `is gone and not recognized. 

I used Maven to pull version seven and it does not recognize those classes. 

-update:
it does not recognize `TradeMessage tradeMessage = (TradeMessage) streamMessage;` either.

New update:
The problem was on my side; it seems the two libraries were mixing up and it was using the other one. It is now fixed. Thank you for the prompt reply.
